x-pack/solutions/security/plugins/cloud_security_posture/server/plugin.ts

```diff
@@ -161,6 +167,26 @@ export class CspPlugin
           }
         );
 
+        plugins.fleet.registerExternalCallback(
+          'packagePolicyPostUpdate',
+          async (
+            packagePolicy: PackagePolicy,
+            soClient: SavedObjectsClientContract,
+            esClient: ElasticsearchClient
+          ): Promise<PackagePolicy> => {
+            if (!getIsNewIndexAliasCreated()) {
```

Comment:
the naming of this utility causing confusion here. On one hand it looks like you already checked that new index alias is created, but then inside you check it again. This is cofusing without knowing that getIsNewIndexAliasCreated actually check for transform to be present. What about making it expicit then and name `getIsTransformAPartOfIntegration` or smth